src/zjs_buffer.c

```diff
@@ -89,7 +90,7 @@ static jerry_value_t zjs_buffer_write_bytes(const jerry_value_t this,
         return zjs_error("zjs_buffer_write_bytes: invalid argument");
     }
 
-    uint32_t value = jerry_get_number_value(argv[0]);
+    uint32_t value = (uint32_t)(int32_t)jerry_get_number_value(argv[0]);
```

Comment:
@kenchris, this broke our big endian functions when testing with unsigned int over 2bil like 0xdeadbeef as we do in tests/test-buffer-rw.js. Then I noticed in the Node docs that behavior is undefined if you pass them anything other than an unsigned number (i.e. a negative). So the right solution is to fix your demo probably. But I think I do have a way to make it continue to work. I'll cc you on the PR shortly if you want to try it.